src/Goerli-DssSpell.sol

```diff
@@ -31,47 +31,34 @@ contract DssSpellAction is DssAction {
     // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO -q -O - 2>/dev/null)"
     string public constant override description = "Goerli Spell";
 
-    // Gelato Keeper Testing Contracts
-    address constant TOP_UP1 = 0xB77827FabA70185D1EaD053648f665d2C41f7906;
-    address constant TOP_UP2 = 0xbe77Cd403Be3F2C7EEBC3427360D3f9e5d528F43;
-
-    // Math
-    uint256 constant MILLION  = 10 ** 6;
-
-    // Turn off office hours
+    // Turn on office hours
     function officeHours() public override returns (bool) {
-        return false;
+        return true;
     }
 
     function actions() public override {
+        address MCD_CLIP_USDT_A = DssExecLib.getChangelogAddress("MCD_CLIP_USDT_A");
+
+        // Offboard USDT-A
+        // https://vote.makerdao.com/polling/QmRNwrTy?network=mainnet#vote-breakdown
+
+        // 1 million DAI maximum liquidation amount
+        DssExecLib.setIlkMaxLiquidationAmount("USDT-A", 1_000_000);
+
+        // flip breaker to enable liquidations
+        DssExecLib.setValue(MCD_CLIP_USDT_A, "stopped", 0);
 
-        // Adjusting Auction Parameters for ETH-A, ETH-B, ETH-C, and WBTC-A
-        // https://vote.makerdao.com/polling/QmfGk3Dm?network=mainnet#poll-detail
+        // authorize breaker
+        DssExecLib.authorize(MCD_CLIP_USDT_A, DssExecLib.clipperMom());
 
-        // Decrease the Auction Price Multiplier (buf) for ETH-A, ETH-C, and WBTC-A vaults from 1.3 to 1.2
-        DssExecLib.setStartingPriceMultiplicativeFactor("ETH-A",  12000);
-        DssExecLib.setStartingPriceMultiplicativeFactor("ETH-C",  12000);
-        DssExecLib.setStartingPriceMultiplicativeFactor("WBTC-A", 12000);
+        // breaker at a 5% drop
+        DssExecLib.setLiquidationBreakerPriceTolerance(MCD_CLIP_USDT_A, 9500);
```

Comment:
This should be removed, the tolerance was already set to this value.

Reply:
good catch, redundant.